dsgrid_project/dimensions/supplemental/residential_end_uses.csv

```diff
@@ -0,0 +1,47 @@
+id,name,fuel_id,unit
+electricity_bath_fan,Bath Fan,electricity,MWh
+electricity_ceiling_fan,Ceiling Fan,electricity,MWh
+electricity_clothes_dryer,Clothes Dryer,electricity,MWh
+electricity_clothes_washer,Clothes Washer,electricity,MWh
+electricity_cooking_range,Cooking Range,electricity,MWh
+electricity_cooling,Cooling,electricity,MWh
+electricity_dishwasher,Dishwasher,electricity,MWh
+electricity_exterior_holiday_lighting,Exterior Holiday Lighting,electricity,MWh
+electricity_extra_refrigerator,Extra Refrigerator,electricity,MWh
+electricity_fans,Fans,electricity,MWh
```

Comment:
Did you intend to add this row? It is in commercial_end_uses.csv and was not previously here.